### PR TITLE
Fixed syntax error

### PR DESCRIPTION
**Pull request by cdenneen**
```Error: Invalid tag "::php::fpm::pool" at /.../php/manifests/fpm.pp:51 on node ...```

### CODE REVIEW

File: manifests/fpm.pp [hunk 1]
```diff
@@ -48,7 +48,7 @@
   anchor { '::php::fpm::end': }
 
   $real_pools = hiera_hash('php::fpm::pools',  $pools)
-  create_resources(::php::fpm::pool, $real_pools)
+  create_resources('php::fpm::pool', $real_pools)
```
Comment (fpletz) [inline manifests/fpm.pp]: Does `'::php::fpm::pool'` also solve this problem for you? It's generally better and faster to use the fully qualified form to refer to classes and resources.

Comment (cdenneen) [inline manifests/fpm.pp]:
```puppet
  $real_pools = hiera_hash('php::fpm::pools',  $pools)
  create_resources('::php::fpm::pool', $real_pools)
```
results in:
Error: Invalid tag "::php::fpm::pool"

```puppet
  $real_pools = hiera_hash('php::fpm::pools',  $pools)
  create_resources('php::fpm::pool', $real_pools)
```
results in:
no error

Comment (globin) [inline manifests/fpm.pp]: Which puppet version are you on?

Comment (cdenneen) [inline manifests/fpm.pp]: Some legacy and some current (3.2.3 and 3.8.3)

**Comment by fpletz** [comment]
This is a problem with Puppet 3.2.x. See #129.

**Pull request closed without merging**